DE51509 - add tooltip-position property to d2l-navigation-button-icon

derekmoran opened the pull request:
[DE51509](https://rally1.rallydev.com/#/42960320374ud/custom/236803223728?detail=%2Fdefect%2F675164753105): Tool tip for iterating between students in CE is cut off

Pairs with this PR:
* https://github.com/Brightspace/consistent-evaluation/pull/1631

![image](https://github.com/BrightspaceUILabs/navigation/assets/126340096/381079c2-c589-4e79-a1c6-2f11b017df4d)

derekmoran commented:
> Visual diff tests failed - pull request #198 has been opened with the updated goldens.

😬

dlockhart commented:
Haha yeah those visual-diffs are because Chrome change since the last time they ran. I'll put up an empty PR to get those changes in first, then you can rebase on top of that.

dlockhart commented on README.md:
```diff
@@ -168,6 +168,7 @@ Add the `d2l-navigation-main-footer` component, and provide elements for the `ma
 | `icon` | String | Preset icon key (e.g. `tier1:gear`) |
 | `no-highlight-border` | Boolean | Visually hides the highlight border when hovered/focused |
 | `text-hidden` | Boolean | Visually hides the text |
+| `tooltip-location` | String | Location of the tooltip ( top\|bottom\|left\|right ); default is bottom |
```
Can you call this `tooltip-position` so it's consistent with the tooltip attribute name?

dlockhart commented:
I'm having some actual issues with the visual-diffs failing that I need to spend some time investigating.

derekmoran commented:
> Haha yeah those visual-diffs are because Chrome change since the last time they ran. I'll put up an empty PR to get those changes in first, then you can rebase on top of that.

Thank you very much Dave, as always you rock sir 🤘
